### DISCUSSION TITLE
Permission to use /papi should default to op (and have an associated permissions node)

### DISCUSSION THREAD

**Discussion by SXRWahrheit** [PlaceholderAPI/PlaceholderAPI#746] [(3) Ideas]
### Confirmation

- [X] I checked the Issues and Pull request tab for any existing issues or PRs.
- [X] My Feature Request is for **PlaceholderAPI** and not any expansion or other plugin.

### Type

New config option

### Description

The `/papi` command can be viewed in tab complete and ran by players by default. It does not appear to have an explicit permissions node associated with it. Even with permissions denied for `/plugins` etc., the fact that PAPI is installed can be determined by the presence of the command in the tab list, and the version can be viewed by players.

As a matter of best informational security practices, by default, only operators should be able to run `/papi`, see it in the tab list, etc.

**Comment by Andre601** [comment]
I for my part can't see why hiding plugins is a good security practice.
The only "reason" I see is to make it harder for people to copy the server, which imo is stupid.
Because if they bother to copy a server down to its plugins do they have dangerously low standards for the most part.
